blade-graphics/src/vulkan/mod.rs

```diff
@@ -23,6 +23,7 @@ struct RayTracingDevice {
 struct Workarounds {
     extra_sync_src_access: vk::AccessFlags,
     extra_sync_dst_access: vk::AccessFlags,
+    intel_fix_descriptor_pool_leak: bool,
```

Comment:
Perhaps we can instead store it as
```
extra_descriptor_pool_create_flags: vk::DescriptorPoolCreateFlags
```
Or at least call it something neutral, like `force_free_descriptor_set_flag`